Lib/test/test_random.py

```diff
@@ -163,6 +163,24 @@ def test_sample_on_sets(self):
             population = {10, 20, 30, 40, 50, 60, 70}
             self.gen.sample(population, k=5)
 
+    def test_sample_on_seqsets(self):
+        class SeqSet(abc.Sequence, abc.Set):
+            def __init__(self, items=()):
+                self._items = list(dict.fromkeys(items, None))
```

Comment:
Why not just `self._items = items` and pass a list with unique items?

Reply:
Habit (to ensure the correct invariant), you're right that it's completely unnecessary for something which only exists for the purpose of a test.